FrontEnd/src/UserAuthentication/SignUpPage.jsx

```diff
@@ -0,0 +1,48 @@
+import  { useNavigate } from "react-router-dom"
+import {useState} from "react";
+import {registerUser} from "./Auth";
+
+function SignUpPage(){
+    let navigate = useNavigate();
+    const [signUpEmail, setSignUpEmail] = useState('');
+    const [signUpPassword, setSignUpPassword] = useState('');
+    const minimumPasswordLength = 6;
```

Comment:
Since this is a constant value, you should capitalize it and can move it above line 5.
```suggestion
    const MIN_PASSWORD_LENGTH = 6;
```